libs/components/tabs/testing/src/modules/tabs/tab-button-harness.ts

```diff
@@ -93,6 +101,16 @@ export class SkyTabButtonHarness extends SkyComponentHarness {
     return await (await this.#getTabButton()).hasClass('sky-btn-tab-disabled');
   }
 
+  /**
+   * Whether the tab button is in a Wizard tabset.
+   */
+  public async isInWizardTabset(): Promise<boolean> {
+    return (
+      (await (await this.#getTabButton()).getAttribute('aria-labelledby')) ===
```

Comment:
Would `return await (await this.#getTabButton()).hasClass('sky-btn-tab-wizard');` work better here?